src/core_plugins/kibana/public/management/sections/objects/components/objects_table/objects_table.js

```diff
@@ -0,0 +1,314 @@
+    const { savedObjectsClient } = this.props;
+    const { selectedSavedObjects } = this.state;
+    const objects = await savedObjectsClient.bulkGet(selectedSavedObjects);
+    await retrieveAndExportDocs(objects.savedObjects, savedObjectsClient);
+  }
+
+  onExportAll = async () => {
+    const { kbnIndex, $http } = this.props;
+    const results = await scanAllTypes($http, kbnIndex, INCLUDED_TYPES);
+    saveToFile(JSON.stringify(flattenDeep(results.hits), null, 2));
+  }
+
+  finishImport = () => {
+    this.hideImportFlyout();
+    this.fetchSavedObjects();
+    this.fetchCounts();
+  }
+
+  showImportFlyout = () => {
+    this.setState({ isShowingImportFlyout: true });
+  }
+
+  hideImportFlyout = () => {
+    this.setState({ isShowingImportFlyout: false });
+  }
+
+  onDelete = async () => {
+    const { savedObjectsClient } = this.props;
+    const { selectedSavedObjects } = this.state;
+
+    const indexPatterns = selectedSavedObjects.filter(object => object.type === 'index-pattern');
+    if (indexPatterns.length) {
+      await this.props.indexPatterns.cache.clearAll();
+    }
+
+    const objects = await savedObjectsClient.bulkGet(selectedSavedObjects);
+    const deletes = objects.savedObjects.map(object => savedObjectsClient.delete(object.type, object.id));
+    await Promise.all(deletes);
+
+    // Unset this
+    this.setState({ selectedSavedObjects: [] });
+
+    // Fetching all data
+    await this.fetchSavedObjects();
+    await this.fetchCounts();
+  }
+
+  getRelationships = async (type, id) => {
+    return await getRelationships(type, id, this.props.$http, this.props.basePath);
+  }
+
+  renderFlyout() {
+    if (!this.state.isShowingImportFlyout) {
+      return null;
+    }
+
+    return (
+      <Flyout
+        close={this.hideImportFlyout}
+        done={this.finishImport}
+        services={this.props.services}
+        indexPatterns={this.props.indexPatterns}
+        newIndexPatternUrl={this.props.newIndexPatternUrl}
+      />
+    );
+  }
+
+  renderRelationships() {
+    if (!this.state.isShowingRelationships) {
+      return null;
+    }
+
+    return (
+      <Relationships
+        id={this.state.relationshipId}
+        type={this.state.relationshipType}
+        title={this.state.relationshipTitle}
+        getRelationships={this.getRelationships}
+        close={this.onHideRelationships}
+        getDashboardUrl={this.props.getDashboardUrl}
+      />
+    );
+  }
+
+  render() {
+    const {
+      selectedSavedObjects,
+      page,
+      perPage,
+      savedObjects,
+      totalItemCount,
+      isSearching,
+      savedObjectCounts,
+    } = this.state;
+
+    const selectionConfig = {
+      itemId: 'id',
+      onSelectionChange: this.onSelectionChanged,
+    };
+
+    const filterOptions = INCLUDED_TYPES.map(type => ({
+      value: type,
+      name: type,
+      view: `${type} (${savedObjectCounts[type]})`,
+    }));
+
+    return (
+      <div style={{ padding: '0 1rem' }}>
```

Comment:
Why `1rem`? Seems random and thus maybe we could just wrap this in a `<Fragment>`, thereby avoiding unnecessary CSS?

Reply:
It came from a comment from @jen-huang and this number felt right to me. I agree it feels a bit arbitrary but I guess it really is. Pinging @elastic/kibana-design